PSix/Array.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Array.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/29/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+
+extension Array {
+    
+    static func filterNils(array: [T?]) -> [T] {
```

Comment:
Did you:
[] Add `pod 'dollar'` to your podfile
[] Add `use_frameworks!` to your pofile
[] Use cocoapods 0.36.1

?